index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title> Prospex </title>
+    <meta name=“viewport” content=“width=device-width, initial-scale=1”>
+    <Link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <Link a href="prospex.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <section id="logo-nav">
+      <img src="prospex.png">
+    </section>
+
+    <section class="flex-container" id="section-nav">
+      <ul>
+        <li> <a href="#who-we-are"> Who we are </a>
```

Comment:
Unorthodox use of space between your list items and your anchors. It does look clean and tidy though! Good work :)